src/kvilib/irc/KviIrcMask.cpp

```diff
@@ -29,40 +29,40 @@
 /*
 	@doc: irc_masks
 	@title:
-		Irc masks
+		IRC masks
 	@type:
 		generic
 	@short:
-		Decription of the standard IRC masks
+		Description of the standard IRC masks
 	@keyterms:
 		irc masks, nickname, username, hostname, wildcard
 	@body:
 		[big]Simple masks[/big][br]
-		An irc mask is a string in a special format that identifies an user on irc.[br]
+		An IRC mask is a string in a special format that identifies an user on IRC.[br]
 		The standard basic format is:[br]
 		[b]<nick>!<username>@<host>[/b][br]
 		The <nick> part contains the nickname with that the user is widely known across the network.[br]
-		The nickname format is generally restricted by the irc network rules: usually it has a maximum
+		The nickname format is generally restricted by the IRC network rules: usually it has a maximum
 		length (9 on actual IrcNet servers for example), and can contain only a defined set of characters.
 		Just as example, the character '!' obviously can't be included in a nickname.[br]
 		The <username> part is the machine username of the remote user: this is usually
```

Comment:
it is not *usually*. it is. You can't change an ident (unless you abuse hack your IRCd... not that... anyone has.... or anything....)